test/utils/DateFormatter.test.ts

```diff
@@ -0,0 +1,57 @@
+import { expect } from 'chai';
+
+import { dateToString } from 'app/utils/DateFormatter';
+
+describe('DateFormatter', () => {
+
+    it('should throw invalid date error', () =>{
+
+        const date: Date = new Date('nonsense date value');
+        expect(() => dateToString(date)).throws('DateFormatter - Invalid date');
+
+    });
+
+    it('should return YYYY-MM-DD format for a local date', () =>{
+
+        const date: Date = new Date('2020-10-30T23:00:00.000');
+        expect(dateToString(date)).to.equal('2020-10-30');
+
+    });
+
+    it('should return YYYY-MM-DD format for a UTC date', () =>{
```

Comment:
Not sure what you testing here. Dates are the same cause in October GMT is being used in UK which has the same offset as UTC.

Reply:
Its a small little detail that probably isnt worth testing, but notice the "z" at the end of the string:
`new Date('2020-10-30T23:00:00.000z'`

this denotes that the timezone is zero utc offset. I guess in this case im making sure that format works, but I agree thats its not necessary